cypress/integration/examples/hometask.e2e.js

```diff
@@ -1,11 +1,11 @@
 describe("Cypress e2e testing", () => {
-    it('should open page', () => {
+    it('should test app', () => {
         cy.visit('http://localhost:4000/');
 
         cy.title().should('include', "ReactJS App");
     });
 
-    it('should has value', () => {
+    it('elements should has some values', () => {
```

Comment:
Let me recommend one technique to verify the quality of test descriptions - values in all `describe`s and `it`s should reveal application's features.

```
Cypress e2e testing
  should test app
  elements should has some values
```
Considering above, how much do we know about the application?

Reply:
Same technique can be applied to unit tests - "test report" should reveal units' (components') capabilities.

Reply:
> Let me recommend one technique to verify the quality of test descriptions - values in all `describe`s and `it`s should reveal application's features.
> 
> ```
> Cypress e2e testing
>   should test app
>   elements should has some values
> ```
> Considering above, how much do we know about the application?

Done